Different process indicator

EagleEye25 opened the pull request.

Commit:
added processing indicator on buy flow

Commit:
change position of indicator

Commit:
minor change

Commit:
html improvement

Commit:
change process indicator on buy flow

Commit:
implemented modal processing indicator

EagleEye25 commented:
@AllienWorks would you please do some styling changes on the modal for us :D

Commit:
fix tests

Commit:
Processing modal: styled

Commit:
fixed Travis SCSS warnings

AllienWorks commented:
Modal styled.

Also fixed Travis SCSS warnings. Codebase test still failing though, due to:
```
Chromium 62.0.3202 (Ubuntu 0.0.0) ProcessingModalComponent should create FAILED
	Uncaught NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'ng:///DynamicTestModule/ProcessingModalComponent.ngfactory.js'.
Chromium 62.0.3202 (Ubuntu 0.0.0): Executed 58 of 174 (1 FAILED) (0 secs / 51.812 secs)
Chromium 62.0.3202 (Ubuntu 0.0.0) ProcessingModalComponent should create FAILED
	Uncaught NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'ng:///DynamicTestModule/ProcessingModalComponent.ngfactory.js'.
```

vikas-cis committed:
test case fixed

vikas-cis committed:
trigger build

Commit:
Reverts out unnecessary code.

Commit:
Fixes merge conflicts
merge branch 'market-beta' of https://github.com/particl/particl-desktop into different-process-indicator

Commit:
Removes unnecessary processingStatus variable usage.
